language/move-core/types/Cargo.toml

```diff
@@ -32,6 +32,7 @@ proptest = "1.0.0"
 proptest-derive = "0.3.0"
 regex = "1.5.5"
 serde_json = "1.0.64"
+arbitrary = "1.1.7"
```

Comment:
why the change?

Reply:
I was getting weird failures on local test where the package was not found. This seemed to fix it. I pushed it to see if CI will complain